js/piwik.js

```diff
@@ -7618,10 +7655,16 @@ if (typeof window.Piwik !== 'object') {
              * Call this method once the user has given consent. This will cause all tracking requests from this
              * page view to be sent. Please note that the given consent won't be remembered across page views. If you
              * want to remember consent across page views, call {@link rememberConsentGiven()} instead.
+             *
+             * It will also automatically enable cookies if they were disabled previously.
+             *
+             * @param bool [enableCookies=true] Internal parameter. Defines whether cookies should be enabled or not.
```

Comment:
we could generally expose this parameter as a support official parameter. However, then we'd also need to add this to `rememberConsentGiven ()` method and the problem is that we'd then need to persist in yet another cookie whether `rememberConsentGiven ()` should always enable cookies or not. Easier to consider it an internal parameter for now